LICENSE

```diff
@@ -1,7 +1,14 @@
-Copyright 2019 Factomize LLC
+Copyright 2019-2020 Factomize LLC
 
-Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:

```

Comment:
I'd suggest use of MIT or other acknowledged, reviewed, and community accepted license rather than rolling your own.  Of course, integration with factomd would be slightly easier with MIT

Reply:
This is the MIT license